ibm/mas_devops/roles/suite_app_install/vars/defaultspecs/predict.yml

```diff
@@ -1,10 +1,7 @@
 ---
 # Default application specs for Predict
 mas_app_spec:
-  bindings:
-    jdbc: "{{ mas_app_bindings_jdbc }}"
   components: {}
```

Comment:
the jdbc binding is not needed anymore?

Reply:
Yes, it is not needed on main operator. It is needed on ws operator